x-pack/plugins/security_solution/public/common/components/paginated_table/index.tsx

```diff
@@ -303,12 +303,14 @@ const PaginatedTableComponent: FC<SiemTables> = ({
               </EuiFlexItem>
 
               <PaginationWrapper grow={false}>
-                <EuiPagination
-                  data-test-subj="numberedPagination"
-                  pageCount={pageCount}
-                  activePage={myActivePage}
-                  onPageClick={goToPage}
-                />
+                {totalCount > 0 && (
```

Comment:
Shouldn't we check `pageCount > 1` here? I mean, is there any point in showing the pagination if there is only one page? Or am I missing something?

Reply:
I said nothing, I checked this is the common behavior everywhere :)

Reply:
Ok, hehehe. Just for context:
This bug was happening in most tables. When `pageCount` is zero `EuiPagination` shows the next page button 🤷 
It is explained on the documentation: 

https://elastic.github.io/eui/#/navigation/pagination
> If the total number of pages cannot be accurately determined, you can pass 0 as the pageCount. This will remove the button numbers and rely solely on the arrow icon buttons for navigation. Without a total page count, the last page button will pass back -1 for the activePage.